pkg/importer/data-processor_test.go

```diff
@@ -311,9 +311,12 @@ var _ = Describe("Resize", func() {
 			url: url,
 		}
 		dp := NewDataProcessor(mdp, "dest", "dataDir", "scratchDataDir", "", 0.055, false)
```

Comment:
For consistency, it might be better to use the constant DefaultGlobalOverhead instead of hard code the `0.055`.